src/main/core/manager.rs

```diff
@@ -7,11 +7,13 @@ use std::sync::Arc;
 use std::time::Duration;
 
 use anyhow::{self, Context};
+use atomic_refcell::AtomicRefCell;
 use rand::seq::SliceRandom;
 use rand_xoshiro::Xoshiro256PlusPlus;
 
 use crate::core::controller::{Controller, ShadowStatusBarState, SimController};
 use crate::core::scheduler::runahead::Runahead;
+use crate::core::scheduler::scheduler::Scheduler;
```

Comment:
I agree, but I'll be renaming the second scheduler module once I add in a thread-per-core scheduler, so I'm going to leave this for now so that I don't need to move it back to a new file later.